### ISSUE TITLE
[BUG] Multiplayer variables showing up as plain text

### ISSUE THREAD

**Issue by zndrmn** [hypherionmc/simple-rpc-public#59]
**Describe the bug**
Multiplayer variables showing up as plain text on first join, after backing out and rejoining will correct the variable. And %servericon% wasn't working either so I used a server override entry for my server.
Similar to #46

**To Reproduce**
1. Join any server (some work fine)
2. Check discord status
3. Observe discord status

**Expected behavior**
Variable showing proper player numbers

**Screenshots**
Crime Evidence
![image](https://user-images.githubusercontent.com/77690119/228144149-4b7de911-e042-4bac-9b6d-a7a55bc229b5.png)

**Versions (DO NOT USE LATEST! BE SPECIFIC!):**
 - OS: win11pro
 - JAVA Version: 17.0.3
 - Mod Version: 3.2.2
 - Discord Version: 184038 
 - Minecraft Version: 1.19.3
 - Forge/Fabric Version: Fabric

**Additional context**
My server uses a couple of mods that are required in order for the client to join.

**Comment by hypherionmc** [comment]
> Multiplayer variables showing up as plain text on first join, after backing out and rejoining will correct the variable.

This issue seems to be caused by some servers themselves. I use the mod myself when playing mutiplayer across forge/fabric/vanilla servers and can't say I've had this happen once. I have seen some other players experience this (people I see on discord), but no one ever reports it (aside from the linked issue you mentioned, which was fixed already).

> And >%servericon% wasn't working either so I used a server override entry for my server.

This only works on servers that are publicly reachable across the internet and that can be queried by services like https://mcsrvstat.us/. If these services don't show an icon, nether will the mod, since this is the exact service it uses to display the icon.

> Also it messes with this too, it's now showing JarFix's logo lmao

This has nothing to do with the mod. That section is controlled entirely by discord, so it's likely detecting JarFix as a running application and showing that logo instead

**Comment by hypherionmc** [comment]
Damn github. Can't edit. To answer the main issue, I'll have to somehow reproduce this myself so I can add a fix for it

**Comment by zndrmn** [comment]
now that I think about it the reason why the image wasn't showing up is proll because I'm directly connecting to it with just 0 as the ip
Directly connecting with 0 as the ip could just be messing with things in general

**Comment by zndrmn** [comment]
yeah this is how I connect to my server (which is hosted on the same pc I play on)
<img src="https://user-images.githubusercontent.com/77690119/228162843-b7954657-e2b2-4baf-8021-72d14754baca.png" width="378" height="461">

**Comment by hypherionmc** [comment]
I see. That makes sense then.

I'll take a look at improving the system for resolving the variables, so that if one fails, they don't all fail. I'll keep you updated
